### PR TITLE
Multiple visual and collision elements load properly as aggregates

### PR DESCRIPTION
**Pull request by cpaxton**

### CODE REVIEW

File: src/urdf_parser_py/urdf.py [hunk 1]
```diff
@@ -352,18 +352,19 @@ def joint_type(self, value): self.type = value
 class Link(xmlr.Object):
     def __init__(self, name=None, visual=None, inertial=None, collision=None,
                  origin=None):
+        self.aggregate_init()
         self.name = name
-        self.visual = visual
-        self.inertial = inertial
-        self.collision = collision
+        self.visuals = []
+        self.inertials = inertial
```
Comment (sloretz) [inline src/urdf_parser_py/urdf.py:358]: Since a link may only have one `<interial>` tag I think this should stay as `intertial`

**Review by sloretz — CHANGES_REQUESTED** [review]
Thanks for the pull request @cpaxton! This allows parsing `<link>` with [multiple collision or visual](http://wiki.ros.org/urdf/XML/link). Would you mind retargeting this at the `melodic-devel` branch? The changes may break existing code in indigo and kinetic.

**Comment by cpaxton** [comment]
No problem, will close and retarget.

File: src/urdf_parser_py/urdf.py (hunk 1 above)
Comment (cpaxton) [reply src/urdf_parser_py/urdf.py:358]: You mean inertial correct?

**Pull request closed without merging**